libs/datasets/timeseries.py

```diff
@@ -1289,13 +1322,43 @@ def combined_datasets(
             timeseries_dfs.append(field_wide_df.loc[(slice(None), selected_location_id), :])
             location_id_so_far = location_id_so_far.union(selected_location_id).sort_values()
             provenance_series.append(
-                datasets[dataset_name].provenance.loc[selected_location_id, slice(field)]
+                datasets[dataset_name].provenance.loc[selected_location_id, field]
             )
-    output_wide = pd.concat(timeseries_dfs, verify_integrity=True)
+
+    for field, datasets_list in static_field_dataset_source.items():
+        static_column_so_far = None
+        for dataset_name in datasets_list:
+            dataset_column = datasets[dataset_name].static.get(field)
+            if dataset_column is None:
+                continue
+            dataset_column = dataset_column.dropna()
+            assert dataset_column.index.names == [CommonFields.LOCATION_ID]
+            if static_column_so_far is None:
+                static_column_so_far = dataset_column
+            else:
+                selected_location_id = dataset_column.index.difference(static_column_so_far.index)
+                static_column_so_far = pd.concat(
+                    [static_column_so_far, dataset_column.loc[selected_location_id]],
+                    sort=True,
+                    verify_integrity=True,
+                )
+        static_series.append(static_column_so_far)
+    output_timeseries_wide_dates = pd.concat(timeseries_dfs, verify_integrity=True)
+    assert output_timeseries_wide_dates.index.names == [PdFields.VARIABLE, CommonFields.LOCATION_ID]
+    assert output_timeseries_wide_dates.columns.names == [CommonFields.DATE]
+    if output_timeseries_wide_dates.empty:
+        output_timeseries_wide_variables = _EMPTY_TIMESERIES_WIDE_VARIABLES_DF
+    else:
+        output_timeseries_wide_variables = (
```

Comment:
I'm not sure I understand `wide_variables` here?

Reply:
Add some comments.